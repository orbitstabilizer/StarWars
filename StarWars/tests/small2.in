1
Sector1 REPUBLIC
40
Officer officer1 COMMAND 1
Officer officer2 COMMAND 2
Officer officer3 GUNNERY 3
Officer officer4 GUNNERY 4
Officer officer5 PILOTING 5
Officer officer6 PILOTING 6
Officer officer7 ENGINEERING 7
Officer officer8 ENGINEERING 8
Officer officer9 TACTICAL 9
Officer officer10 TACTICAL 10
Officer officer11 COMMAND 10
Officer officer12 COMMAND 10
Officer officer13 GUNNERY 10
Officer officer14 GUNNERY 10
Officer officer15 PILOTING 10
Officer officer16 PILOTING 10
Officer officer17 ENGINEERING 10
Officer officer18 ENGINEERING 10
Officer officer19 TACTICAL 1
Officer officer20 TACTICAL 1
Sith freeSith1 10 10 10
Sith freeSith2 10 10 10
Sith freeSith3 10 10 10
Sith freeSith4 10 10 10
Sith freeSith5 10 10 10
Sith freeSith6 10 10 10
Sith freeSith7 10 10 10
Sith freeSith8 10 10 10
Sith sith1 10 10 10
Jedi jedi1 10 10 10
Officer officer21 COMMAND 1
Officer officer22 COMMAND 1
Officer officer23 GUNNERY 1
Officer officer24 GUNNERY 1
Officer officer25 PILOTING 1
Officer officer26 PILOTING 1
Officer officer27 ENGINEERING 1
Officer officer28 ENGINEERING 1
Officer officer29 TACTICAL 1
Officer officer30 TACTICAL 1
2
SeparatistBattleship Battleship1 1 1
1 29
RepublicCruiser Cruiser1 1 2
1 30
60
40 1 1
40 2 1
40 3 1
40 4 1
40 5 1
40 6 1
40 7 1
40 8 1
40 9 1
40 10 1
40 11 1
40 12 1
40 13 1
40 14 1
40 15 1
40 16 1
40 17 1
40 18 1
40 19 1
40 20 1
40 21 1
40 22 1
40 23 1
40 24 1
40 25 1
40 26 1
40 27 1
40 28 1
40 1 2
40 2 2
40 3 2
40 4 2
40 5 2
40 6 2
40 7 2
40 8 2
40 9 2
40 10 2
40 11 2
40 12 2
40 13 2
40 14 2
40 15 2
40 16 2
40 17 2
40 18 2
40 19 2
40 20 2
40 31 2
40 32 2
40 33 2
40 34 2
40 35 2
40 36 2
40 37 2
40 38 2
40 39 2
40 40 2
41 1 1
41 1 2