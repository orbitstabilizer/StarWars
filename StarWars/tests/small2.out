Warship Battleship1 in (Sector1, 1)
freeSith1 324022
Warship Cruiser1 in (Sector1, 2)
jedi1 88596
Sith freeSith1 is in Battleship1
60
Sith freeSith2 is in Battleship1
60
Sith freeSith3 is in Battleship1
60
Sith freeSith4 is in Battleship1
60
Sith freeSith5 is in Battleship1
60
Sith freeSith6 is free
60
Sith freeSith7 is free
60
Sith freeSith8 is free
60
Sith sith1 is in Battleship1
60
Jedi jedi1 is in Cruiser1
50
Officer officer10 is in Battleship1
TACTICAL 10
Officer officer11 is in Battleship1
COMMAND 10
Officer officer12 is in Cruiser1
COMMAND 10
Officer officer13 is in Battleship1
GUNNERY 10
Officer officer14 is in Cruiser1
GUNNERY 10
Officer officer15 is in Battleship1
PILOTING 10
Officer officer16 is in Cruiser1
PILOTING 10
Officer officer17 is in Battleship1
ENGINEERING 10
Officer officer18 is in Cruiser1
ENGINEERING 10
Officer officer9 is in Battleship1
TACTICAL 9
Officer officer8 is in Battleship1
ENGINEERING 8
Officer officer7 is in Battleship1
ENGINEERING 7
Officer officer6 is in Battleship1
PILOTING 6
Officer officer5 is in Battleship1
PILOTING 5
Officer officer4 is in Battleship1
GUNNERY 4
Officer officer3 is in Battleship1
GUNNERY 3
Officer officer2 is in Battleship1
COMMAND 2
Officer officer1 is free
COMMAND 1
Officer officer19 is in Cruiser1
TACTICAL 1
Officer officer20 is in Cruiser1
TACTICAL 1
Officer officer21 is in Cruiser1
COMMAND 1
Officer officer22 is in Cruiser1
COMMAND 1
Officer officer23 is in Cruiser1
GUNNERY 1
Officer officer24 is free
GUNNERY 1
Officer officer25 is free
PILOTING 1
Officer officer26 is free
PILOTING 1
Officer officer27 is free
ENGINEERING 1
Officer officer28 is free
ENGINEERING 1
Officer officer29 is free
TACTICAL 1
Officer officer30 is free
TACTICAL 1
